umu/umu_run.py

```diff
@@ -63,6 +63,9 @@ def parse_args() -> Namespace | tuple[str, list[str]]:  # noqa: D103
         ),
         formatter_class=RawTextHelpFormatter,
     )
+    parser.add_argument("-v", "--version", action='store_true',
+                        help="show the version and exit"
```

Comment:
Nit: `show this version and exit`

This makes it consistent with the help message.